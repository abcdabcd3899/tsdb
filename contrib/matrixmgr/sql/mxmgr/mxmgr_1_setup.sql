-- cleanup
--
-- start_ignore
DROP DATABASE IF EXISTS matrixmgr;
-- end_ignore

CREATE DATABASE matrixmgr;

\c matrixmgr

--
-- setup
--

-- should fail due to no matrixts
CREATE EXTENSION matrixmgr;

-- should success
CREATE EXTENSION matrixmgr CASCADE;
