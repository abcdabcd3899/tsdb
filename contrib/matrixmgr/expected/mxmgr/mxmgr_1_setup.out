-- cleanup
--
-- start_ignore
DROP DATABASE IF EXISTS matrixmgr;
-- end_ignore
CREATE DATABASE matrixmgr;
\c matrixmgr
--
-- setup
--
-- should fail due to no matrixts
CREATE EXTENSION matrixmgr;
ERROR:  required extension "matrixts" is not installed
HINT:  Use CREATE EXTENSION ... CASCADE to install required extensions too.
-- should success
CREATE EXTENSION matrixmgr CASCADE;
NOTICE:  installing required extension "matrixts"
